DuckDuckGoTests/AppUrlsTests.swift

```diff
@@ -23,6 +23,76 @@ import XCTest
 class AppUrlsTests: XCTestCase {
     
     var mockStatisticsStore = MockStatisticsStore()
```

Comment:
You never use this instantiation because you insantiate again in setup. Choose one or the other.

Reply:
thanks! I meant to change that